### PR TITLE
Search for civic address is lower cased on all searches now

### PR DESCRIPTION
**Pull request by urmikgov**

### CODE REVIEW

File: services/apps/alcs/src/portal/inbox/notice-of-intent/inbox-notice-of-intent.service.ts [hunk 1]
```diff
@@ -155,8 +155,8 @@ export class InboxNoticeOfIntentService {
     }
 
     if (searchDto.civicAddress) {
-      query = query.andWhere('parcel.civic_address like :civic_address', {
-        civic_address: `%${searchDto.civicAddress}%`,
+      query = query.andWhere('LOWER(parcel.civic_address) like LOWER(:civic_address)', {
+        civic_address: `%${searchDto.civicAddress}%`.toLowerCase(),
```
Comment (dhaselhan) [inline services/apps/alcs/src/portal/inbox/notice-of-intent/inbox-notice-of-intent.service.ts:159]: question: If you are lower casing it in the SQL with LOWER, why do you also lowercase it first?

File: services/apps/alcs/src/alcs/search/search.controller.ts [hunk 2]
```diff
@@ -288,7 +288,8 @@ export class SearchController {
       !searchDto.dateDecidedTo &&
       !searchDto.resolutionNumber &&
       !searchDto.resolutionYear &&
-      !isStringSetAndNotEmpty(searchDto.legacyId);
+      !isStringSetAndNotEmpty(searchDto.legacyId) &&
+      !isStringSetAndNotEmpty(searchDto.civicAddress);
```
Comment (dhaselhan) [inline services/apps/alcs/src/alcs/search/search.controller.ts:292]: question: Why are you changing this? I don't see it mentioned in the MR

Comment (urmikgov) [reply services/apps/alcs/src/alcs/search/search.controller.ts]: had to remove it from mr

File: services/apps/alcs/src/portal/inbox/notice-of-intent/inbox-notice-of-intent.service.ts (hunk 1 above)
Comment (urmikgov) [reply services/apps/alcs/src/portal/inbox/notice-of-intent/inbox-notice-of-intent.service.ts:159]: just  consistency with all fields

Comment (dhaselhan) [reply services/apps/alcs/src/portal/inbox/notice-of-intent/inbox-notice-of-intent.service.ts:159]: Sorry, I'm not sure what you mean, consistency with which fields?

File: services/apps/alcs/src/portal/inbox/notification/inbox-notification.service.ts [hunk 3]
```diff
@@ -146,10 +146,11 @@ export class InboxNotificationService {
     }
 
     if (searchDto.civicAddress) {
-      query = query.andWhere('parcel.civic_address like :civic_address', {
-        civic_address: `%${searchDto.civicAddress}%`,
+      query = query.andWhere('LOWER(parcel.civic_address) like LOWER(:civic_address)', {
+        civic_address: `%${searchDto.civicAddress}%`.toLowerCase(),
       });
     }
+    console.log(query)
```
Comment (dhaselhan) [inline services/apps/alcs/src/portal/inbox/notification/inbox-notification.service.ts:153]: Left over debug log

File: services/apps/alcs/src/portal/inbox/notice-of-intent/inbox-notice-of-intent.service.ts (hunk 1 above)
Comment (dhaselhan) [reply services/apps/alcs/src/portal/inbox/notice-of-intent/inbox-notice-of-intent.service.ts:159]: I noticed you resolved this without answer my question, did you miss it?

Comment (urmikgov) [reply services/apps/alcs/src/portal/inbox/notice-of-intent/inbox-notice-of-intent.service.ts:159]: @dhaselhan i am just keeping it consistent with other searches, can remove it from everywhere

**Review by dhaselhan — APPROVED** [review]
LGTM 👍

**Pull request merged**